templates/skeleton/public/readme.md

```diff
@@ -0,0 +1,7 @@
+Use the `/public` folder to store static files that **shouldn't be processed at build time**.
```

Comment:
This file here will be available in `domain.com/readme.md` if they don't remove it manually. Perhaps we should inform about assets in the root template readme instead?

Reply:
Ha, I wondered about that. Could move it out for sure, although I think having it in context has some value. Is there some way to configure an ignore rule?

Reply:
Nm, apparently there's a time in the future where this will change, and our proxy logic will not filter assets like this, so then this file would start to show up.

Reply:
> is the desire to give this folder context on Github with the readme file? Or also to give context inside vscode after you've scaffolded a new project?

More the latter. Why ask them to google it or pore over the docs when we could just say what the folder is, right in context? Seems to me like the canonical role of a readme file.

That said, `example.com/readme.md` is not a desirable outcome 😅 

Is there some exclude/ignore list we can give to Vite at build time?

Reply:
Oh yeah, this is not urgent at all